# Configuration Settings for C94-M8P device

debug: 1                    # Levels 0-4 (0 means no debug statements will print)

device: /dev/ttyACM0
frame_id: gps_base
ublox_version: 8
baudrate: 19200             # C94-M8P specific
uart_in: 0                  # No UART in for base
uart_out: 32                # RTCM 3
rate: 4                     # in Hz, must be at least 1 Hz
nav_rate: 4                 # in number of measurement cycles, should be 1 Hz

dynamic_model: stationary   # Velocity restricted to 0 m/s. Zero dynamics 
                            # assumed.
fix_mode: auto              # Options: auto, 2d, 3d
dr_limit: 0                 # Dead-reckoning

# TMODE3 Config
tmode3: 1                   # Survey-In Mode
svin_reset: false           # True: disables and re-enables survey-in (resets)
                            # False: Disables survey-in only if TMODE3 is  
                            # disabled
sv_in_min_dur: 300          # Survey-In Minimum Duration [s]
sv_in_acc_lim: 3.0          # Survey-In Accuracy Limit [m]

# RTCM out config
rtcm_ids: [5, 87, 77, 230]  # RTCM Messages to configure for Base station
                            # Enabled: GPS MSM7, GLONASS MSM7, GLONASS CP bias,
                            # Stationary RTK ref
                            # 0xF5 0x05 Stationary RTK reference station ARP
                            # 0xF5 0x4A GPS MSM4
                            # 0xF5 0x4D GPS MSM7
                            # 0xF5 0x54 GLONASS MSM4
                            # 0xF5 0x57 GLONASS MSM7
                            # 0xF5 0x7C BeiDou MSM4
                            # 0xF5 0x7F BeiDou MSM7
                            # 0xF5 0xE6 GLONASS code-phase biases
                            # 0xF5 0xFE Reference station PVT 
                            #(u-blox proprietary RTCM Message)
rtcm_rates: [1, 1, 1, 10]   # in number of navigation solutions

# GNSS Config
enable_glonass: true        # Supported by C94-M8P
enable_beidou: false        # Supported by C94-M8P
enable_qzss: false          # Supported by C94-M8P
enable_sbas: false          # Not supported by C94-M8P
enable_ppp: false

# Message subscriptions
all: true
inf: true
aid_hui: false